docs/medical-api/api-reference/fhir/consolidated-data-query-post.mdx

```diff
@@ -77,18 +81,35 @@ that will contain a URL to retrieve the data from.
 
 ## Response
 
-Returns a JSON containing a `status` property indicating the query status. Possible values for consolidated
-data query status are `processing`, `completed`, or `failed`.
+Returns an array of `queries` with the following fields: `requestId`, `status`, `startedAt`, `dateFrom`, `dateTo`, `resources`, and `conversionType`.
```

Comment:
See `StartConsolidatedQueryProgressResponse`:
- It returns a JSON
- The JSON contains a single `query`, not an array

I'd also add a bit more context as to what the property `query` represents.

Thinking about a suggested text, I actually think we should return this:

```ts
type StartConsolidatedQueryProgressResponse = ConsolidatedQuery;
```

Why:
1. That's the actual representation of the initiated query, including the ID (`requestId`);
2. We don't use `message` AFAIK, so might as well remove it.

Then, my suggestion for explaining the response would be something like:

```
Returns the filters used for this query, in addition to the identifier used by the system to uniquely identify this query (property `requestId`).
```

And list the properties and a small text explaining what each represent.